tests/infrastructure/test_ialirt_database_stack.py

```diff
@@ -0,0 +1,62 @@
+"""Test the IAlirt database stack."""
+
+import boto3
+import pytest
+from boto3.dynamodb.conditions import Key
+from moto import mock_dynamodb
+
+TABLE_NAME = "imap-packetdata-table"
+
+
+@pytest.fixture()
+def dynamodb():
+    """Initialize DynamoDB resource and create table."""
+    with mock_dynamodb():
+        dynamodb = boto3.resource("dynamodb", region_name="us-west-2")
+        table = dynamodb.create_table(
+            TableName=TABLE_NAME,
+            KeySchema=[
+                # Partition key
+                {"AttributeName": "sct_vtcw_reset#sct_vtcw", "KeyType": "HASH"},
+            ],
+            AttributeDefinitions=[
+                {"AttributeName": "sct_vtcw_reset#sct_vtcw", "AttributeType": "S"},
+            ],
+            BillingMode="PAY_PER_REQUEST",
+        )
+        yield dynamodb
```

Comment:
can you yield the table itself here? You are connecting to the table in each of the other tests below this.

```suggestion
        yield table
```